packages/mds-daily/api.ts

```diff
@@ -24,7 +24,7 @@ import stream from 'mds-stream'
 import { providers, providerName } from 'mds-providers'
 import areas from 'ladot-service-areas'
 import { UUID, VehicleEvent, Telemetry, CountMap, DeviceID } from 'mds'
-import { VEHICLE_EVENTS, VEHICLE_STATUSES, EVENT_STATUS_MAP } from 'mds-enums' // FIXME replace eventually
+import { VEHICLE_EVENTS, VEHICLE_STATUSES, EVENT_STATUS_MAP } from 'mds-enums' // TODO replace eventually
```

Comment:
I don't see why the enums are something we should replace eventually... Can @karcass weigh in on keeping this comment?

Reply:
Stale comment.  Remove.